VideoLocker/res/drawable/row_discussion_first_comment_background.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
+    android:shape="rectangle">
+    <solid android:color="@color/edx_grayscale_neutral_white" />
+    <stroke
+        android:width="@dimen/edx_hairline"
+        android:color="@color/edx_grayscale_neutral_light" />
+    <corners android:radius="@dimen/edx_box_radius"
```

Comment:
It's per the Android docs, I don't know what API levels are affected:
"Note: Every corner must (initially) be provided a corner radius greater than 1, or else no corners are rounded. If you want specific corners to not be rounded, a work-around is to use android:radius to set a default corner radius greater than 1, but then override each and every corner with the values you really want, providing zero ("0dp") where you don't want rounded corners."

http://developer.android.com/guide/topics/resources/drawable-resource.html#Shape